r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -538,15 +531,6 @@ test_that("extract quarter from date", {
   )
 })
 
-test_that("extract month from date", {
-  compare_dplyr_binding(
-    .input %>%
-      mutate(x = month(date)) %>%
-      collect(),
-    test_df
-  )
-})
```

Comment:
Did you intend to remove this test? Or maybe I missed that it was being moved somewhere else?

Reply:
Yes, that's intentional. It appeared twice in the file (probably from a bad rebase/merge), and I chose to keep the longer version:
https://github.com/apache/arrow/blob/62e81780dda234106fba5a3148f0d19045d66898/r/tests/testthat/test-dplyr-funcs-datetime.R#L561

If you look in master right now you will see the same test twice: https://github.com/apache/arrow/blob/master/r/tests/testthat/test-dplyr-funcs-datetime.R

Reply:
Aaah, good catch! Thanks